core/src/components/cat-select/cat-select.tsx

```diff
@@ -132,7 +142,7 @@ export class CatSelect {
   /**
    * The value of the select.
```

Comment:
add some more docs here about the types and how they are influenced by the inputs